app/app.module.ts

```diff
@@ -39,6 +39,7 @@ import { ApplicationGroupsModule } from "./applicationgroups/applicationgroups.m
 import   appRoutes  from "./app.routes";
```

Comment:
This is not related to the code, I thought of adding a favicon to the UI.
We can add this favicon in index.html head section: 
<link rel="icon"
          type="image/png"
          href="images/contiv_36x36.png">